UM/TaskManagement/HttpRequestManager.py

```diff
@@ -393,8 +398,9 @@ def _onRequestFinished(self, request_data: "HttpRequestData") -> None:
         # We do nothing if the request was aborted or and error was detected because an error callback will also
         # be triggered by Qt.
         reply = request_data.reply
-        if reply is not None and reply.error() != QNetworkReply.NoError:
-            if reply.error() == QNetworkReply.OperationCanceledError:
+        reply_error = reply.error() # error() must only be called once
```

Comment:
You now call reply.error before checking if reply is not None, this could result in a crash?